appium/webdriver/webdriver.py

```diff
@@ -114,6 +114,43 @@ def _make_w3c_caps(caps: Dict) -> Dict[str, List[Dict[str, Any]]]:
 T = TypeVar('T', bound='WebDriver')
 
 
+class ExtensionBase:
+    """
+    Used to define an extension command.
+
+    TODO:
+    add examples
+    """
+
+    def __init__(self, execute: Callable):
```

Comment:
Python also allows to make typing more precise with Generics, for example Callable[str, str] defines a method, which returns str and accepts one param of type str